vendor/cicicom/s-lg-i3.yaml

```
@@ -0,0 +1,66 @@
+name: SLG-I3 Parking Sensor
+description: Cicicom's LoRaWAN smart parking sensor SL-G-I3 accurately detects parking events, by utilizing cutting-edge triple detection technology (magnetic sensor, motion sensor, radar). All sensors and algorithms are tuned and optimized for ultra-low power consumption, providing continuous vehicle detection with higher than 99% accuracy and resulting to complete monitoring and management of parking spots.
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '1.0'
+    numeric: 1
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: 'V1.4.0'
+    numeric: 1
+    hardwareVersions:
+      - '1.0'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867, RU864-870
+    profiles:
+      EU863-870:
+        # Unique identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: s-lg-i3
+        lorawanCertified: true
+
+# Sensors that this device features (optional)
+# Valid values are:
+# 4-20 ma, accelerometer, altitude, analog input, auxiliary, barometer, battery, button, bvoc, co, co2, conductivity,
+# current, digital input, dissolved oxygen, distance, dust, energy, gps, gyroscope, h2s, humidity, iaq, level, light,
+# lightning, link, magnetometer, moisture, motion, no, no2, o3, particulate matter, ph, pir, pm2.5, pm10, potentiometer,
+# power, precipitation, pressure, proximity, pulse count, pulse frequency, radar, rainfall, rssi, smart valve, snr, so2,
+# solar radiation, sound, strain, surface temperature, temperature, tilt, time, tvoc, uv, vapor pressure, velocity,
+# vibration, voltage, water potential, water, weight, wifi ssid, wind direction, wind speed.
+sensors:
+   - radar, magnetometer
```

Thank you for creating a PR

Please run `make validate fmt` in a command prompt to adhere to our schema. This for example needs to become:
```suggestion
   - radar
   - magnetometer
```